src/features/emails/l10n/messageIds.ts

```diff
@@ -2,10 +2,59 @@ import { m, makeMessages } from 'core/i18n';
 
 export default makeMessages('feat.emails', {
   emailActionButtons: {
+    afterLock: m<{ numTargets: number }>(
+      'Will schedule email for {numTargets} people'
+    ),
+    beforeLock: m('Lock targeting to proceed'),
+    delete: m('Delete'),
     delevery: m('Delivery'),
+    deliveryDate: m('Delivery date'),
+    deliveryTime: m('Delivery time'),
+    duplicate: m('Duplicate'),
+    lockDesc: m(
+      'Make sure the current exact targeting will be used during delivery'
+    ),
+    lockTarget: m('Lock targeting'),
+    schedule: m('Schedule'),
+    sendAway: m('Send away'),
```

Comment:
Is this intended to say "Send _anyway_"?

Reply:
Oops It is "Send anyway" in Figma yeah. I read it wrong..!